sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/implementation/ConfigurationService.java

```diff
@@ -48,6 +48,20 @@ Mono<Response<ConfigurationSetting>> getKeyValue(
         @HeaderParam("If-None-Match") String ifNoneMatch,
         Context context);
 
+    @Get("kv/{key}")
+    @ExpectedResponses({200})
+    @UnexpectedResponseExceptionType(HttpResponseException.class)
+    Response<ConfigurationSetting> getKeyValueSync(
+        @HostParam("url") String url,
+        @PathParam("key") String key,
```

Comment:
I'd suggest to name async operations "Async" and sync without suffix.

Take a look at what autorest does today:

Look for `generate-sync-async-clients` and `sync-methods` https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/anomalydetector/azure-ai-anomalydetector/swagger/README.md

Then
https://github.com/Azure/azure-sdk-for-java/blob/91ecd1062a14b36c553fd37db1cdeb60ffe269eb/sdk/anomalydetector/azure-ai-anomalydetector/src/main/java/com/azure/ai/anomalydetector/AnomalyDetectorClient.java#L61

And then
https://github.com/Azure/azure-sdk-for-java/blob/91ecd1062a14b36c553fd37db1cdeb60ffe269eb/sdk/anomalydetector/azure-ai-anomalydetector/src/main/java/com/azure/ai/anomalydetector/implementation/AnomalyDetectorClientImpl.java#L397-L400

It would be good to get appcofing to the layout that's kind of like what autorest would generate with addition that we don't reactor-block on async.